src/Jumbotron.js

```diff
@@ -2,11 +2,21 @@ import React from 'react';
 import classNames from 'classnames';
 
 const Jumbotron = React.createClass({
+  propTypes: {
+    componentClass: React.PropTypes.node.isRequired
```

Comment:
I think `node` might be the wrong type here...we are really looking for a string, or a function. i.e. tagname or Component Class.